persistence.js

```diff
@@ -330,19 +330,14 @@ function updateWithClientData (that, client, packet, cb) {
   var clientListKey = outgoingKey + client.id
   var packetKey = 'packet:' + packet.brokerId + ':' + packet.brokerCounter
 
+  var ttl = that.packetTTL(packet)
```

Comment:
Actually we should probably read the TTL of the original key here and use it instead of computing it anew, but I wanted to be consistent with the TTL being set to `clientUpdateKey` few lines bellow and didn't want to fix to different issues in a single PR.